system/jlib/jptree.cpp

```diff
@@ -1772,6 +1772,67 @@ void PTree::addPropInt64(const char *xpath, __int64 val)
     }
 }
 
+void PTree::setPropReal(const char * xpath, double val)
```

Comment:
looks ok, but I expected it to be implemented as something like
```
{
    std::string s = std::to_string(val);
    setProp(xpath, s.c_str());
}
```
would that be better?

Reply:
it would have same net effect, but calling setProp would involve a virtual and some unecessary checking and an extra strlen call, ultimately to get to same setLocal call being called directly here - so this is more efficient.